system/nxplayer/nxplayer.c

```diff
@@ -121,6 +125,14 @@ static const struct nxplayer_ext_fmt_s g_known_ext[] =
 
 static const int g_known_ext_count = sizeof(g_known_ext) /
                     sizeof(struct nxplayer_ext_fmt_s);
+
+static const struct nxplayer_dec_ops_s g_dec_mp3 =
```

Comment:
can we make the similar adjust for .wav files?

Reply:
done